app/src/main/java/com/lambdaschool/hackathon_portal/ui/MainActivity.kt

```diff
@@ -77,6 +77,10 @@ class MainActivity : AppCompatActivity() {
                     nav_host_fragment.findNavController().navigate(R.id.settingsFragment)
                     title = "Settings"
                 }
+
+                R.id.nav_drawer_add_hackathon -> {
+                    nav_host_fragment.findNavController().navigate(R.id.addHackathonFragment)
```

Comment:
My oversight on PR #12. Forgot to refactor these calls.

navController is injected on line 40 and can be called directly

```suggestion
                    navController.navigate(R.id.addHackathonFragment)
```

Reply:
Changed in commit 3b1138a1fc3b243d79ce1c20727eac01c3287d03